[2025-08-16 23:00:18,930] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_orders manual__2025-08-16T23:00:08.759833+00:00 [queued]>
[2025-08-16 23:00:18,941] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_orders manual__2025-08-16T23:00:08.759833+00:00 [queued]>
[2025-08-16 23:00:18,942] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 23:00:18,943] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-16 23:00:18,944] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-16 23:00:18,956] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): load_orders> on 2025-08-16 23:00:08.759833+00:00
[2025-08-16 23:00:18,961] {standard_task_runner.py:52} INFO - Started process 159 to run task
[2025-08-16 23:00:18,971] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '1_load_initial_data', 'load_orders', 'manual__2025-08-16T23:00:08.759833+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/initialise_data.py', '--cfg-path', '/tmp/tmp65yr6mdd', '--error-file', '/tmp/tmpfy7rtx9w']
[2025-08-16 23:00:18,977] {standard_task_runner.py:80} INFO - Job 15: Subtask load_orders
[2025-08-16 23:00:19,065] {task_command.py:369} INFO - Running <TaskInstance: 1_load_initial_data.load_orders manual__2025-08-16T23:00:08.759833+00:00 [running]> on host 03b58f608368
[2025-08-16 23:00:19,192] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=1_load_initial_data
AIRFLOW_CTX_TASK_ID=load_orders
AIRFLOW_CTX_EXECUTION_DATE=2025-08-16T23:00:08.759833+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-16T23:00:08.759833+00:00
[2025-08-16 23:00:19,194] {sql.py:262} INFO - Executing: COPY dbt_raw_data.orders FROM '/sample_data/orders_v1.csv' DELIMITER ';' CSV HEADER;
[2025-08-16 23:00:19,218] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 23:00:19,280] {logging_mixin.py:115} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/connection.py:339 DeprecationWarning: The "schema" arg has been renamed to "database" as it contained the database name.Please use "database" to set the database name.
[2025-08-16 23:00:19,313] {base.py:68} INFO - Using connection ID 'dbt_postgres_instance_raw_data' for task execution.
[2025-08-16 23:00:19,323] {sql.py:375} INFO - Running statement: COPY dbt_raw_data.orders FROM '/sample_data/orders_v1.csv' DELIMITER ';' CSV HEADER;, parameters: None
[2025-08-16 23:00:24,025] {sql.py:384} INFO - Rows affected: 500000
[2025-08-16 23:00:24,079] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=1_load_initial_data, task_id=load_orders, execution_date=20250816T230008, start_date=20250816T230018, end_date=20250816T230024
[2025-08-16 23:00:24,142] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-16 23:00:24,176] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
